gdal/frmts/idrisi/IdrisiDataset.cpp

```diff
@@ -2815,6 +2815,10 @@ CPLErr IdrisiGeoReference2Wkt( const char* pszFilename,
     {
         oSRS.SetSinusoidal( dfCenterLong, dfFalseEasting, dfFalseNorthing );
     }
+    else if( EQUAL( pszProjName, "CylindricalEA" ) || EQUAL( pszProjName, "Cylindrical Equal Area" ))
+    {	
+		oSRS.SetCEA( dfStdP1, dfCenterLong, dfFalseEasting, dfFalseNorthing );
```

Comment:
please change your editor settings or fix manually the indentation to use only space and not tablulations. here and below